Wikipedia/Code/WMFReferencePopoverMessageViewController.m

```diff
@@ -129,4 +123,29 @@ - (BOOL)accessibilityPerformEscape {
     return true;
 }
 
+- (void)applyTheme:(WMFTheme *)theme {
+    self.theme = theme;
+    if (self.viewIfLoaded == nil) {
+        return;
+    }
+
+    self.view.backgroundColor = theme.colors.paperBackground;
+    
+    self.textView.linkTextAttributes = @{NSForegroundColorAttributeName: theme.colors.link};
+    
+    [self.textView setAttributedText:[self attributedStringForHTML:[self referenceHTMLWithSurroundingHTML]]];
```

Comment:
Could things be re-arranged a bit so this call to `setAttributedText` (and the one on 144) could be made after the call to `applyTheme` rather than inside it?